components/RegisterForm.tsx

```diff
@@ -37,7 +39,10 @@ export const RegisterForm = memo<PublicQueryResult>(
 						"Content-Type": "application/json",
 					},
 					body: JSON.stringify({
-						data: state,
+						data: {
+							...state,
+							isUKLanguage: locale === "uk",
```

Comment:
is this used anywhere?

Reply:
tbh not sure.. do you have any idea if we need it elsewhere?